nilearn/_utils/niimg_conversions.py

```diff
@@ -77,6 +78,10 @@ def _iter_check_niimg(niimgs, ensure_ndim=None, atleast_4d=False,
             if not _check_fov(niimg, ref_fov[0], ref_fov[1]):
                 if target_fov is not None:
                     from nilearn import image  # we avoid a circular import
+                    if target_fov == "first":
```

Comment:
How about now?

BTW: The target_fov == "first" does not appear to be working at the moment because ref_fov will just remain None as it appears.